INSTALL.md

```diff
@@ -0,0 +1,78 @@
+Installing Release Builds
+=========================
+
+These instructions explain how to download, install, and build the K packages.
+Current supported systems are:
+
+-   Arch Linux
+-   Ubuntu Bionic (18.04)
+-   Ubuntu Xenial (16.04)
+-   Debian Stretch
+
+Downloading Packages
+--------------------
+
+We release our packages on GitHub, visit the [Releases](https://github.com/kframework/k/releases) page to see available versions.
+Releases are generated as often as possible from the `master` branch of the repository.
+
+Installing Packages
+-------------------
+
+### Ubuntu/Debian
+
+Install the package with (`X.Y.Z` is version number, `ID` is platform identifier):
+
+```sh
+sudo apt install ./kframework_X.Y.Z_amd64_ID.deb
+```
+
+On Debian Stretch, you also must first enable stretch-backports by running:
```

Comment:
we support buster now, not stretch, and on buster this is not true.

Reply:
Removed.